gutenberg.php

```diff
@@ -67,6 +67,9 @@ function the_gutenberg_project() {
  * @since 0.1.0
  */
 function gutenberg_menu() {
+	if ( ! gutenberg_can_edit_post( $post ) ) {
+		return;
+	}
```

Comment:
This check is creating a warning and for me it doesn't make sense to check this here at all. It's being addressed in #12053